src/sdk/namespace_fs.js

```diff
@@ -62,11 +67,11 @@ const dir_cache = new LRUCache({
     name: 'nsfs-dir-cache',
     load: async dir_path => {
         const time = Date.now();
-        const stat = await fs.promises.stat(dir_path);
+        const stat = await nb_native().stat(dir_path);
         let sorted_entries;
         let usage = config.NSFS_DIR_CACHE_MIN_DIR_SIZE;
         if (stat.size <= config.NSFS_DIR_CACHE_MAX_DIR_SIZE) {
-            sorted_entries = await fs.promises.readdir(dir_path, { withFileTypes: true });
+            sorted_entries = await nb_native().readdir(dir_path, /* withFileTypes */ true);
```

Comment:
In next PR I recommend to keep the second arg an object like fs.promises does.

Reply:
Hmm on second thought - do we even need this option??
Isn't it always better to return the file types instead of having this option?

Reply:
I don't see a reason, since this just creates more checks in the native C++ layer and we aren't actually getting any value from it in the NodeJS code.
Do you want me to remove it?

Reply:
I think so. If we later decide we need it we can always introduce it.

Reply:
done